frontend/app/routes/public/renew/$id/ita/update-mailing-address.tsx

```diff
@@ -343,92 +271,12 @@ export default function RenewItaUpdateAddress() {
                   errorMessage={errors?.mailingPostalCode}
                   required={mailingPostalCodeRequired}
                 />
+                <InputCheckbox id="copyMailingAddress" name="copyMailingAddress" value="copy" checked={copyAddressChecked} onChange={checkHandler}>
+                  {t('renew-ita:update-address.home-address.use-mailing-address')}
+                </InputCheckbox>
               </div>
```

Comment:
```suggestion
              </div>
              <InputCheckbox id="copyMailingAddress" name="copyMailingAddress" value="copy" checked={copyAddressChecked} onChange={checkHandler}>
                  {t('renew-ita:update-address.home-address.use-mailing-address')}
              </InputCheckbox>
```

Before:
![image](https://github.com/user-attachments/assets/54350786-c1f4-4e67-b1ab-a759a102fcb7)

After:
![image](https://github.com/user-attachments/assets/ec88a501-5318-4e15-a931-acbb11c57281)